aries-site/src/examples/templates/PageHeader/PageHeader.js

```diff
@@ -1,14 +1,16 @@
 import PropTypes from 'prop-types';
 import { Header, Heading } from 'grommet';
 
-export const PageHeader = ({ title }) => (
-  <Header>
+export const PageHeader = ({ title, actions, ...rest }) => (
+  <Header {...rest}>
     <Heading level={1} margin="none" size="small">
       {title}
     </Heading>
+    {actions}
```

Comment:
curious why you did `actions` and not just have `children` be passed in?

Reply:
I'm imagining a future where there is additional content such as contextual navigation, subtitles, actions, etc. where we might not want to assume that children belong in the actions content area.

For example:
```
<PageHeader 
  title={title} 
  subtitle={subtitle} 
  actions={
    <Box direction="row">
      <Button ... />
      <Button ... />
      <Menu ... />
    </Box>
  }
  contextualNav={
    upButton: {label1, href1},
    breadCrumbs: [
      {label2, href2},
      {label3, href3},
      {label4, href4},
      ...
    ]
  />
--------------------

export const PageHeader = ({
  title,
  subtitle,
  actions,
  contextualNav,
  ...rest
}) => {
  const [upButton, setUpButton] = useState(null);
  const [breadCrumbs, setBreadCrumbs] = useState(null);

  useEffect(() => {
    if (contextualNav) {
      setUpButton(contextualNav.upButton);
      setBreadCrumbs(contextualNav.breadCrumbs);
    }
  }, [contextualNav]);

  return (
    <Header pad={{ top: 'large' }} {...rest}>
      <Box>
        <Box gap="xsmall">
          <>
            {breadCrumbs && <BreadCrumbs links={breadCrumbs} />}
            {upButton && (
              <UpButton
                alignSelf="start"
                icon={<FormPrevious />}
                {...upButton}
                label={upButton.label}
                href={upButton.href}
              />
            )}
          </>
          <Heading level={1} margin="none" size="small">
            {title}
          </Heading>
        </Box>
        {typeof subtitle === 'string' ? (
          <Paragraph margin="none">{subtitle}</Paragraph>
        ) : (
          <>{subtitle}</>
        )}
      </Box>
      {actions}
    </Header>
  );
};

PageHeader.propTypes = {
  actions: PropTypes.oneOfType([PropTypes.node]),
  contextualNav: PropTypes.shape({
    upButton: PropTypes.object,
    breadCrumbs: PropTypes.arrayOf(PropTypes.object),
  }),
  subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
  title: PropTypes.string,
};
``` 

This may not be the API we ultimately want. I could see us potentially going for something like:
```
<PageHeader>
  <PageTitle />
  <PageActions />
  <ContextualNav />
</PageHeader>
```

Will come down to playing with level of opinionatedness(?) and ability to affect layout...

Reply:
got it just wanted to clarify I do like that we have the `actions` Im glad you are thinking ahead 😄 
I also like the idea of having this 

```
<PageHeader>
  <PageTitle />
  <PageActions />
  <ContextualNav />
</PageHeader>
```